src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1275,6 +1275,12 @@
   <data name="DoNotUseStackallocInLoopsMessage" xml:space="preserve">
     <value>Potential stack overflow. Move the stackalloc out of the loop.</value>
   </data>
+  <data name="InstantiateArgumentExceptionsCorrectlyChangeToTwoArgument" xml:space="preserve">
+    <value>Change to call the two argument constructor, pass null for the message.</value>
+  </data>
+  <data name="InstantiateArgumentExceptionsCorrectlyFlipArgumentOrder" xml:space="preserve">
+    <value>Swap the arguments order</value>
```

Comment:
I think this feels too succinct, don't we need something more descriptive? Something that suggests what was wrong in the original code that needs the swap operation.

Reply:
Given these new resource strings are only used in offered code fix, and not any diagnostic, I think your current succinct wording looks fine to me - you can ignore this suggestion.